cmd/client_changefeed.go

```diff
@@ -401,6 +401,18 @@ func newUpdateChangefeedCommand() *cobra.Command {
 			// Fix some fields that can't be updated.
 			info.CreateTime = old.CreateTime
 			info.AdminJobType = old.AdminJobType
+			info.StartTs = old.StartTs
+			info.ErrorHis = old.ErrorHis
+			info.Error = old.Error
+
+			resp, err := applyOwnerChangefeedQuery(ctx, changefeedID, getCredential())
+			// if no cdc owner exists, allow user to update changefeed config
+			if err != nil && errors.Cause(err) != errOwnerNotFound {
+				return err
+			}
+			if err == nil && !strings.Contains(resp, `"state": "stopped"`) {
```

Comment:
```suggestion
			// Note that the correctness of the logic here depends on the return value of `/capture/owner/changefeed/query` interface.
			// TODO: Using error codes instead of string containing judgments
			if err == nil && !strings.Contains(resp, `"state": "stopped"`) {
```

Reply:
rest lgtm